avocado/etc/avocado/avocado.conf

```diff
@@ -84,3 +84,18 @@ skip_broken_plugin_notification = []
 # options or test types).
 # The keyword "@DEFAULT" will be replaced with all available unused loaders.
 loaders = ['file', '@DEFAULT']
+
+[simpletests.status]
+# Regular Expression that will make the test
```

Comment:
Maybe it'd be worth explicitly saying `python regular expression`, maybe even giving there a link to regexp documentation, because this feature is for simple tests, therefor bash-oriented comunity.